index.bs

```diff
@@ -1730,14 +1729,14 @@ and presents methods for <a>blob URL</a> creation and revocation.
 <pre class="idl">
 [Exposed=(Window,DedicatedWorker,SharedWorker)]
 partial interface URL {
-  static DOMString createObjectURL(Blob blob);
+  static DOMString createObjectURL((Blob or MediaSource) obj);
```

Comment:
Is this valid? `MediaSource` is not exposed everywhere `URL` is.

Reply:
Why wouldn't it be "valid" (for whatever definition of valid you're asking about)? You just won't be able to pass in a MediaSource when in a worker, but that doesn't seem that crazy?

I'm not sure how else we'd be writing this in IDL either, since you can't have two overloads of the same method with different Exposed= values for example (perhaps you could somehow work around it with two separate partial interfaces? But if that is a possible way to get different overloads with different Exposed= values, I'm not sure why it isn't just allowed directly).

Reply:
I'm pretty sure we had this discussion before (likely different people) and decided it ought to work. See also `send()` in https://xhr.spec.whatwg.org/#interface-xmlhttprequest.